SIL.Core.Tests/Xml/XmlSerializationHelperTests.cs

```diff
@@ -412,5 +412,15 @@ public void DeserializeAndSerialize_Normal_RoundTripObject(bool dispose)
 				}
 			}
 		}
+
+		[Test]
+		public void SerializeToString_EncodingSpecified_XmlHeaderHasExpectedEncoding()
+		{
+			var testObj = new TestObject("Fred");
+			var result = XmlSerializationHelper.SerializeToString(testObj);
+			Assert.That(result.Contains("encoding=\"utf-16\""), "Baseline test failed");
+			result = XmlSerializationHelper.SerializeToString(testObj, Encoding.UTF8);
```

Comment:
I wanted to use two test cases, but the encodings are static objects,not enums, so they can't be passed as values to the parameter of the test method using TestCase. The first one is really just a baseline/regression to show that utf-8 is not the default. In any case, I was trying to throw something together quick just to see what's going on, since I can no longer run unit tests locally :-(